### ISSUE TITLE
能否加入索引一直在显存中的功能

### ISSUE THREAD

**Issue by hacktmz** [milvus-io/milvus#1954]
很多场景是nq=1的操作，当前milvus nq=500以内cpu跟gpu没有明显的性能区别，也就是无法充分利用gpu。而原生的faiss-gpu API 可以索引都在显存中，搜索速度在当前milvus 10倍以上，对于百万级别项目非常合适

**Comment by JinHai-CN** [comment]
Yes, we will improve the Milvus to support this function in the future.

**Comment by yhmo** [comment]
link to #1949

**Comment by JinHai-CN** [comment]
Duplicated with #3204
